build-logic/conventions/src/main/kotlin/com/squareup/anvil/conventions/KtlintConventionPlugin.kt

```diff
@@ -13,6 +13,18 @@ open class KtlintConventionPlugin : Plugin<Project> {
 
     target.extensions.configure(KtlintExtension::class.java) { ktlint ->
       ktlint.version.set(target.libs.versions.ktlint)
+      val pathsToExclude = listOf(
+        "build/generated",
+        "root-build/generated",
+        "included-build/generated",
+      )
+      ktlint.filter {
+        it.exclude {
+          pathsToExclude.any { excludePath ->
+            it.file.path.contains(excludePath)
+          }
+        }
+      }
```

Comment:
Is this necessary in the context of the fix? Unclear what the related error/issue is currently

Reply:
It's not necessary in the context of fix, but without this I'm getting ktlint issues on generated code when I'm running ktlint locally:

```
Execution failed for task ':gradle-plugin:ktlintGradleTestSourceSetCheck'.
> A failure occurred while executing org.jlleitschuh.gradle.ktlint.worker.ConsoleReportWorkAction
   > KtLint found code style violations. Please see the following reports:
     - /Users/ilyagulya/Projects/Community/anvil/gradle-plugin/build/root-build/reports/ktlint/ktlintGradleTestSourceSetCheck/ktlintGradleTestSourceSetCheck.txt
```